src/app/Fake.DotNet.Testing.NUnit/NUnit3.fs

```diff
@@ -317,7 +317,7 @@ let internal createProcess (setParams : NUnit3Params -> NUnit3Params) (assemblie
     //let processTimeout = TimeSpan.MaxValue // Don't set a process timeout. The timeout is per test.
     
     let path = Path.GetTempFileName()
-    let args = (sprintf "@%s" path)
+    let args = (sprintf "\"@%s\"" path)
```

Comment:
Hm, can you instead use the `Args` class, this way it should work with other special cases of windows escaping horror as well. Also we might want to add a simple test to check that it is properly escaped.